pkg/proxy/dns.go

```diff
@@ -106,17 +105,7 @@ func (dr *dnsRedirect) UpdateRules(wg *completion.WaitGroup) (revert.RevertFunc,
 // Close the redirect.
 func (dr *dnsRedirect) Close(wg *completion.WaitGroup) (revert.FinalizeFunc, revert.RevertFunc) {
 	return func() {
-		for _, rule := range dr.currentRules {
-			for _, dnsRule := range rule.DNS {
-				dnsName := strings.ToLower(dns.Fqdn(dnsRule.MatchName))
-				dnsNameAsRE := matchpattern.ToRegexp(dnsName)
-				DefaultDNSProxy.RemoveAllowed(dnsNameAsRE, fmt.Sprintf("%d", dr.redirect.endpointID))
-
-				dnsPattern := matchpattern.Sanitize(dnsRule.MatchPattern)
-				dnsPatternAsRE := matchpattern.ToRegexp(dnsPattern)
-				DefaultDNSProxy.RemoveAllowed(dnsPatternAsRE, fmt.Sprintf("%d", dr.redirect.endpointID))
-			}
-		}
+		DefaultDNSProxy.UpdateAllowed(dr.redirect.endpointID, nil)
```

Comment:
This could be a separate commit? Is there a meaningful functional change here?

Reply:
It's to match the change in UpdateAllowed's signature. I could make it a separate commit but then the original wouldn't compile. There shouldn't be a functional change, however, as you suggest.

Reply:
OK, no point making a commit that won't compile.